pkg/kvstore/consul.go

```diff
@@ -27,14 +27,16 @@ import (
 
 	consulAPI "github.com/hashicorp/consul/api"
 	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
 const (
 	consulName = "consul"
 
 	// optAddress is the string representing the key mapping to the value of the
 	// address for Consul.
-	optAddress = "consul.address"
+	optAddress         = "consul.address"
+	consulOptionConfig = "consul.config"
```

Comment:
Given that this is a TLS specific configuration file, maybe name it more specifically `consul.tlsconfig`?

Reply:
Thanks @tgraf for you review, yes that make sense.